framework/src/mesh/MooseMesh.C

```diff
@@ -156,9 +161,9 @@ MooseMesh::validParams()
   params.registerBase("MooseMesh");
 
   // groups
-  params.addParamNamesToGroup(
-      "dim nemesis patch_update_strategy construct_node_list_from_side_list patch_size",
-      "Advanced");
+  params.addParamNamesToGroup("dim nemesis patch_update_strategy "
+                              "construct_node_list_from_side_list patch_size use_checkpoint_mesh",
+                              "Advanced");
```

Comment:
it s time someone made real parameter groups here. I ll do it if you dont

Reply:
I'll leave it to you as I don't fully understand some of those advanced parameters.